### PR TITLE
artik055s/audio : fix new toolchain build errors

### PR DESCRIPTION
**Pull request by abhishek-samsung**

### CODE REVIEW

File: external/audiocodec/aacdec/fxp_mul32_c_equivalent.h [hunk 1]
```diff
@@ -77,7 +77,7 @@ extern "C"
     }
 
 
-    __inline  Int32 fxp_mul_16_by_16bb(Int32 L_var1,  Int32 L_var2)
+    static __inline  Int32 fxp_mul_16_by_16bb(Int32 L_var1,  Int32 L_var2)
```
Comment (sunghan-chang) [inline external/audiocodec/aacdec/fxp_mul32_c_equivalent.h:80]: This is the external open source. Is this correct solution? Could you refer and check original source?

**Comment by sunghan-chang** [comment]
@sangam-swami @abhishek-samsung Let's finish this ASAP. Could you share what you check now?

**Comment by abhishek-samsung** [comment]
> @sangam-swami @abhishek-samsung Let's finish this ASAP. Could you share what you check now?

Hello Mr. Chang, we are looking into the external original opensource code now.

**Comment by Taejun-Kwon** [comment]
I brought it from stagefright in AOSP

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**